ui/app/services/capabilities.ts

```diff
@@ -14,14 +15,18 @@ export default class CapabilitiesService extends Service {
   @service declare readonly api: ApiService;
   @service declare readonly namespace: NamespaceService;
 
-  SUDO_PATHS = [
-    'sys/seal',
-    'sys/replication/performance/primary/secondary-token',
-    'sys/replication/dr/primary/secondary-token',
-    'sys/replication/reindex',
-    'sys/leases/lookup/',
-  ];
-  SUDO_PATH_PREFIXES = ['sys/leases/revoke-prefix', 'sys/leases/revoke-force'];
+  // add paths to PATH_MAP constant with friendly key like 'syncDestinations' for example
```

Comment:
I reread this several times and have some suggestions for making it easier to read on the first go.
```suggestion
// Add API paths to the PATH_MAP constant using a friendly key, e.g., 'syncDestinations'.
// Each path should include placeholders for dynamic values, like:
// apiPath`sys/sync/destinations/${'type'}/${'name'}`
// To retrieve a path, use the key (ex: `syncDestinations`) and provide an object whose keys match
// the dynamic segment names ('type', 'name', etc.).
// The values from the object will be inserted into the placeholders,
// and the fully-resolved path string will be returned.```